public/js/lib/syncscroll.js

```diff
@@ -110,6 +110,22 @@ md.use(markdownitContainer, 'success', { render: renderContainer })
 md.use(markdownitContainer, 'info', { render: renderContainer })
 md.use(markdownitContainer, 'warning', { render: renderContainer })
 md.use(markdownitContainer, 'danger', { render: renderContainer })
+md.use(markdownitContainer, 'spoiler', {
+  validate: function (params) {
+    return params.trim().match(/^spoiler\s+(.*)$/)
+  },
+  render: function (tokens, idx) {
+    var m = tokens[idx].info.trim().match(/^spoiler\s+(.*)$/)
+
+    if (tokens[idx].nesting === 1) {
+      // opening tag
+      return '<details><summary>' + md.utils.escapeHtml(m[1]) + '</summary>\n'
```

Comment:
```suggestion
      return '<details><summary>' + md.utils.escapeHtml(m[1]) + '</summary>\n'
```

you should add class `part` and `data-startline`, `data-endline` for calculate syncscroll mapping.
See addPart(tokens, idx) function for more info.